server/shadow/shadow_rdpsnd.c

```diff
@@ -62,7 +62,6 @@ static void rdpsnd_activated(RdpsndServerContext* context)
 	}
 
 	context->SelectFormat(context, i);
-	context->SetVolume(context, 0x7FFF, 0x7FFF);
```

Comment:
Any specific reason to remove this one? Does not look like it is related to the locking issue.